cfme/tests/automate/custom_button/test_import_export.py

```diff
@@ -0,0 +1,124 @@
+            text=fauxfactory.gen_alphanumeric(),
+            hover=fauxfactory.gen_alphanumeric(),
+            display_for="Single and list",
+            system="Request",
+            request="InspectMe",
+        )
+        if obj_type == "PROVIDER":
+            obj = provider
+        else:
+            obj = appliance.provider_based_collection(provider).all()[0]
+        gp_buttons[obj_type] = [gp, button, obj]
+
+    yield gp_buttons
+
+    for button_group in gp_buttons.values():
+        grp_, button_, _ = button_group
+        button_.delete_if_exists()
+        grp_.delete_if_exists()
+
+
+def checks(obj_type_conf):
+    for obj_type, conf in obj_type_conf.items():
+        gp, button, obj = conf
+        obj.browser.refresh()  # before start checks refresh browser
+        assert gp.exists
+        assert button.exists
+
+        view = navigate_to(button, "Details")
+        assert view.text.text == button.text
+        assert view.hover.text == button.hover
+
+        for destination in ["All", "Details"]:
+            # Note: For VM, custom button not display on All page but only VM page.
+            nav_obj = obj.parent if destination == "All" else obj
+            if obj_type == "VM_INSTANCE" and destination == "All":
+                destination = "VMsOnly"
+
+            view = navigate_to(nav_obj, destination)
+            custom_button_group = Dropdown(view, gp.hover)
+            assert custom_button_group.is_displayed
+            assert custom_button_group.has_item(button.text)
+
+
+@pytest.mark.uncollectif(lambda appliance: appliance.version < "5.10")
+def test_custom_button_import_export(appliance, setup_groups_buttons):
+    """ Test custom button display on a targeted page
+
+    prerequisites:
+        * Appliance with Provider
+        * Custom buttons and groups for some objects type
+
+    Steps:
+        * Check custom buttons and groups available or not
+        * Check for custom buttons in respective implementation location
+        * Export created custom buttons using rake command
+            `rake evm:export:custom_buttons -- --directory /tmp/custom_buttons`
+        * Clean all created buttons and groups
+        * Check properly clean up or not
+        * Import exported custom button yaml file using import rake command
+            `rake evm:import:custom_buttons -- --source /tmp/custom_buttons`
+        * Check for custom buttons and groups which was exported comes back to UI or not
+        * Check for custom buttons in respective implementation location
+    """
+
+    # Check all buttons, groups and respective display at respective locations
+    checks(setup_groups_buttons)
+
+    # Export custom buttons
+    dir_ = appliance.ssh_client.run_command("mkdir /tmp/custom_buttons")
+    assert dir_.success
+    appliance.ssh_client.run_command("vmdb")
+    export = appliance.ssh_client.run_command(
+        "cd /var/www/miq/vmdb/; rake evm:export:custom_buttons -- --directory /tmp/custom_buttons"
```

Comment:
After running command  - appliance.ssh_client.run_command("vmdb") , 
directory changes to vmdb .
No need to cd again to cd /var/www/miq/vmdb/.

Reply:
aahhh my bad; I forgot to remove the above line. @sshveta actually when we fire new command its dirt will treat as `\root` only.

Reply:
:+1:

Reply:
Please remove appliance.ssh_client.run_command("vmdb") in that case